anomaly_detector/model/sompy_model.py

```diff
@@ -0,0 +1,39 @@
+
+from .base_model import BaseModel
+
+import os
+import numpy as np
+import logging
+import sompy
+
+_LOGGER = logging.getLogger(__name__)
+
+class SOMPYModel(BaseModel):
+    "SOMPY alternative SOM implementation with parallelization"
+
+    def train(self, inp, map_size, iterations):
+        mapsize = [map_size, map_size]
+        som = sompy.SOMFactory.build(inp, mapsize, mask=None, mapshape='planer',
+                                     lattice='rect', normalization='var',
+                                     initialization='pca', neighborhood='gaussian',
+                                     training='batch', name='sompy')
+
+        som.train(n_job=1)
```

Comment:
Make `n_job` configuratble. Based on documentation:

> n_job: number of jobs to use to parallelize the traning

https://github.com/sevamoo/SOMPY/blob/master/sompy/sompy.py#L376

I guess we want to run it in parallel, ideally based on CPU count set for a container, but I would at least start with a variable and a parameter in OpenShift template (which would then be used in both - this env var and CPU requests)